diff --git a/target/product/handheld_product.mk b/target/product/handheld_product.mk
index dde952e4fd..c0c6c3c46f 100644
--- a/target/product/handheld_product.mk
+++ b/target/product/handheld_product.mk
@@ -23,17 +23,7 @@ $(call inherit-product, $(SRC_TARGET_DIR)/product/media_product.mk)
 # /product packages
 PRODUCT_PACKAGES += \
     AppStore \
-    Auditor \
-    Calendar \
-    Camera \
-    Contacts \
-    DeskClock \
-    ExactCalculator \
-    Gallery2 \
-    InfoApp \
     LatinIME \
-    Music \
-    PdfViewerGOS \
     preinstalled-packages-platform-handheld-product.xml \
     SettingsIntelligence \
     ThemePicker \
diff --git a/target/product/handheld_system.mk b/target/product/handheld_system.mk
index aa20c63ff6..15b6725650 100644
--- a/target/product/handheld_system.mk
+++ b/target/product/handheld_system.mk
@@ -101,5 +101,9 @@ PRODUCT_PACKAGES += framework-audio_effects.xml
 
 PRODUCT_VENDOR_PROPERTIES += \
     ro.carrier?=unknown \
-    ro.config.notification_sound?=OnTheHunt.ogg \
-    ro.config.alarm_alert?=Alarm_Classic.ogg
+    ro.config.alarm_alert?=Alarm_Classic.ogg \
+    ro.config.notification_sound?=fasten_seatbelt.ogg \
+    ro.config.ringtone?=Ring_Classic_02.ogg
+
+ PRODUCT_COPY_FILES += \
+     frameworks/base/data/bootanimation.zip:system/media/bootanimation.zip \
